instance/factories.py

```diff
@@ -131,6 +149,8 @@ def production_instance_factory(**kwargs):
         "demo_test_users": [],
         # Disable certificates process to reduce load on RabbitMQ and MySQL
         "SANDBOX_ENABLE_CERTIFICATES": False,
+        # HACK: This is a temporary change for BB-2558. It will be removed after upgrading MongoDB to SCRAM.
+        "FORUM_MONGO_AUTH_MECH": ":mongodb_cr",
```

Comment:
@Agrendalath, @xitij2000, this particular change has to be merged before we start doing the Ocim Juniper upgrades in the next sprint (CC @pomegranited ) as the forum errors out without this being set on the Juniper instances.

Reply:
@lgp171188, @pomegranited described this really nicely [here](https://tasks.opencraft.com/browse/BB-2558?focusedCommentId=160445&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-160445).
That's why I asked for FF review here.